src/components/FilterBar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { PropsType } from "../containers/FilterBarContainer";
+import * as objAssign from "object-assign";
+import { Store } from "azsearchstore";
+import { defaultCss } from "../utils/css";
+
+export type State = {};
+
+class FilterBar extends React.Component<PropsType, State> {
+  render() {
+    const { onClear, hasSelectedFacets } = this.props;
+    const text = "clear filter(s)";
+
+    return (
+      <div className="text-right">
+        { hasSelectedFacets ?
+          <a
+            href="#"
+            onClick={ e => {
+              e.preventDefault();
+              if (onClear) onClear();
+            }}
+          >{text}</a>
+        :
+          <span
+          className="text-muted"
```

Comment:
Can you add an entry for these classNames in [src/utils/css.ts](https://github.com/EvanBoyle/AzSearch.js/blob/master/src/utils/css.ts). In order to allow others to customize/theme the controls as they see fit, I've tried to make CSS configurable wherever possible.